deployments.json

```diff
@@ -1350,5 +1350,10 @@
     "name": "Form Testnet",
     "chainId": 132902,
     "url": "https://explorer.form.network/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract"
+  },
+  {
+    "name": "MO Chain",
+    "chainId": 7924,
+    "url": "https://mainnet.mochain.app/address/0xcA11bde05977b3631167028862bE2a173976CA11"
```

Comment:
Sorry, I missed this PR. Can you fix the merge conflicts and verify the contract at that address?